test.md

```diff
@@ -0,0 +1 @@
+This file should be ignored by git.
```

Comment:
## Review Resolved for adding test.md in .gitignore

[@tresorsawasawa](https://github.com/tresorsawasawa), I have done as requested. 
- Kindly let me know why the requirements say to have the add test.md and then put it in the gitignore file. 
- Since it says to add in a seperate commit. Once it is commited I don't think it's possible to ignore it. But thank you for your guidance it really helped.
- Although, it's better to create it and then after addint it to gitignore make that commit

Reply:
@michael-duke I recommend you research the `git -rm --cached -r` command, it might be useful for future situations 😉

Reply:
🥇 love this collaboration @vechicin thank you for the command I will check it and share it